src/main/include/Constants.h

```diff
@@ -27,134 +30,158 @@
 // #define MOD_BUDDY
 //  When buddy climber is installed, uncomment the above line
 
-namespace CONSTANTS {
+namespace CONSTANTS
+{
+
+    // An additive threshold (+/- value) that checks if 2 values (target & source)
+    // are within a range A template so it can be used with units. Call it by:
+    // CONSTANTS::IN_THRESHOLD<type>(x,y,z)
+    template <typename T>
+    static bool IN_THRESHOLD(T source, T target, T range)
+    {
+        return (source >= target - range && source <= target + range);
+    }
+
+    // Change this to true when in match. Disables PID reload on enable
+    constexpr bool IN_MATCH = false;
+
+    namespace CORAL
+    {
 
-// An additive threshold (+/- value) that checks if 2 values (target & source)
-// are within a range A template so it can be used with units. Call it by:
-// CONSTANTS::IN_THRESHOLD<type>(x,y,z)
-template <typename T> static bool IN_THRESHOLD(T source, T target, T range) {
-  return (source >= target - range && source <= target + range);
-}
+        static const std::string LIMELIGHT_ID = "limelight-aft";
+        constexpr units::meters_per_second_t APPROACH_SPEED = 0.25_mps;
 
-  namespace CORAL {
+    }
 
-      static const std::string LIMELIGHT_ID = "limelight-aft";
-      constexpr units::meters_per_second_t APPROACH_SPEED = 0.25_mps; 
+    namespace WRIST
+    {
+        // TODO: Set this to correct ID
```

Comment:
You don't need to do this (and honestly I don't think you should), but if you *really* don't want to forget a todo, add `#pragma message("...")` to add an info message to the compiler output.